### PR TITLE
Release 0.12.4

### PR DESCRIPTION
**Pull request by cbgbt**
*Description of changes:*
This change `cargo update`s dependencies and prepares the repository to release a new crate.

*Testing Done:*
`make integ` tests pass

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by cbgbt** [comment]
Force pushed a commit which fixes the clippy warnings failing in CI.

**Comment by cbgbt** [comment]
^ Fixes a rebasing issue in the previous force push

**Comment by cbgbt** [comment]
Ideally the last push, but apparently the clippy checker is incremental, so a few more lints were caught in CI after I caught the previous. These should be fixed now.

File: tough/src/schema/key.rs [hunk 1]
```diff
@@ -33,7 +33,7 @@ use std::str::FromStr;
 ///  * `Rsa`: PUBLIC is in PEM format and a string. All RSA keys must be at least 2048 bits.
 ///  * `Ed25519`: PUBLIC is a 64-byte hex encoded string.
 ///  * `Ecdsa`: PUBLIC is in PEM format and a string.
-#[derive(Debug, Clone, Deserialize, Serialize, PartialEq)]
+#[derive(Debug, Clone, Deserialize, Serialize, Eq, PartialEq)]
```
Comment (zmrow) [inline tough/src/schema/key.rs:36]: I'm assuming this was a clippy thing?

**Review by zmrow — APPROVED** [review]
:money_with_wings:

File: tough/src/schema/key.rs (hunk 1 above)
Comment (cbgbt) [reply tough/src/schema/key.rs:36]: Yeah, they seem to have added a lint to suggest you derive `Eq` in some cases where you are already deriving `PartialEq`

**Review by rpkelly — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**